### PR TITLE
Using safeState on usePlaceOrder

### PR DESCRIPTION
**Pull request by alfetopito**
As title states.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr215--dexreact.review.gnosisdev.com

File: src/hooks/usePlaceOrder.ts [hunk 1]
```diff
@@ -22,7 +23,7 @@ interface Result {
 }
 
 export const usePlaceOrder = (): Result => {
-  const [isSubmitting, setIsSubmitting] = useState(false)
+  const [isSubmitting, setIsSubmitting] = useSafeState(false)
   const { userAddress } = useWalletConnection()
 
   const placeOrder = useMemo((): ((params: PlaceOrderParams) => Promise<boolean>) => {
```
Comment (Velenir) [inline src/hooks/usePlaceOrder.ts]: Why isn't this a `useCallback`?

Comment (alfetopito) [reply src/hooks/usePlaceOrder.ts]: Now it is :)

File: src/hooks/usePlaceOrder.ts [hunk 2]
```diff
@@ -22,11 +23,11 @@ interface Result {
...
 
 export const usePlaceOrder = (): Result => {
-  const [isSubmitting, setIsSubmitting] = useState(false)
+  const [isSubmitting, setIsSubmitting] = useSafeState(false)
   const { userAddress } = useWalletConnection()
 
-  const placeOrder = useMemo((): ((params: PlaceOrderParams) => Promise<boolean>) => {
-    return async ({ buyAmount, buyToken, sellAmount, sellToken }: PlaceOrderParams): Promise<boolean> => {
+  const placeOrder = useCallback(
+    () => async ({ buyAmount, buyToken, sellAmount, sellToken }: PlaceOrderParams): Promise<boolean> => {
```
Comment (Velenir) [inline src/hooks/usePlaceOrder.ts]: Almost
```suggestion
   async ({ buyAmount, buyToken, sellAmount, sellToken }: PlaceOrderParams): Promise<boolean> => {
```

and no `placeOrder()` on L79

Comment (alfetopito) [reply src/hooks/usePlaceOrder.ts]: Aha! I thought `useCallback` needed to have 0 params. Updated.

**Pull request merged**